src/App.js

```diff
@@ -14,11 +16,105 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const preparedGoods = goodsFromServer.map((good, index) => (
+  {
+    name: good,
+    id: index,
+  }
+));
+
+export class App extends Component {
+  state = {
+    isVisible: true,
+    goods: preparedGoods,
```

Comment:
```suggestion
  state = {
    isHidden: true,
    goods: preparedGoods,
```

у тебе ж список прихований спочатку

Reply:
fix this